primer/brand
# Add Icon Component

## Summary

Part one of adding the background around the icon inside the Bento. This adds the Icon component.

## List of notable changes:

- Added `Icon` Component to behave similar to `Card.Icon` component
- To come: Update `Bento` to accept new `Icon` component.

## What should reviewers focus on?

- `Icon` should match `Card.Icon` visually

## Steps to test:



1. View `Icon` component in storybook
2. Compare to `Card.Icon`
3. Examine API to verify it matches other components.

## Supporting resources (related issues, external links, etc):

- Towards https://github.com/primer/brand/issues/773
- Related https://github.com/github/marketing-platform-services/issues/3553

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">



 </td>
<td valign="top">



</td>
</tr>
</table>
